api/metadata.py

```diff
@@ -2075,12 +2081,10 @@ def metadata_attr_list(self, request):
 
             cursor.close()
             db.close()
-
+            request_finished.send(self)
             return MetadataAttrList(items=data, count=len(data))
 
         except (IndexError, TypeError):
-            if cursor: cursor.close()
-            if db: db.close()
             raise endpoints.InternalServerErrorException('Error retrieving attribute list')
         finally:
             if cursor: cursor.close()
```

Comment:
`if db: db.close()` is called in the `finally` clause in  line 2087. Am I misunderstanding your line note, @s-paquette?

Actually now that I'm looking at it it should be `if db and db.open: db.close()`.